pallets/omnipool/src/lib.rs

```diff
@@ -272,6 +268,7 @@ pub mod pallet {
 			asset_out: T::AssetId,
 			amount_in: Balance,
 			amount_out: Balance,
+			asset_fee: Permill,
```

Comment:
i think it would be better to just add calculated fee and protocol fee here, since we are doing changes to this event, also which amount should be fee deducted from, naming could clear this

Reply:
yes makes sense. this will be needed for the change that staking needs too. 

however this require math to change and to return the value.

Reply:
mby you can just change when the event is emitted - somewhere where you have all numbers in scope

Reply:
to be clear, i dont want to be permill, but balance